### ISSUE TITLE
TEI vs. homegrown schemas

### ISSUE THREAD

**Issue by zme1** [obdurodon/dh_course#129]
We may have gone over this in class and I could just google it, but what are the pros and cons of using TEI compliant markup? Some groups are using it, some aren't. What's the benefits of using it? Just that it's more standardized?

**Comment by zme1** [comment]
The major benefit of using TEI compliant markup is that TEI serves as a standard, such that if someone else in the future were to go on github and look at your code and potentially use it to do a related research project and have part of the work done for them (for example, this past summer when I was doing a research project I found a version of the new testament on github that already had the verses and chapters marked which saved me quite a bit of time), they would know the markup to some extent going in (assuming they were familiar with TEI standards, of course.) Moreover, the required header in TEI forces you to remember that you need to have the origin of the document in your code.
The reason you may not want to use TEI syntax is that it limits you. The beauty of XML is that you can markup whatever you want to, however you want to (essentially), but if you choose to use TEI, you no longer have that much freedom. Depending what you want to do, this can be a minor or a major drawback.

**Comment by zme1** [comment]
Yes to everything Alathia said, and one more dimension: it isn't necessarily either/or. For one project I wanted the greater simplicity of my own schema during development, but I then needed to process my documents with a tool that understood TEI, but not my schema. As a "best of both worlds" strategy, I encoded using my own schema, used XSLT to transform my markup to TEI markup, and then fed the TEI documents to the external application. My schema was quite simple, and the XSLT to do the conversion to TEI took just a couple of hours to write, test, fine-tune, etc.
